src/Test/Utilities/DiagnosticAnalyzerTestBase.cs

```diff
@@ -88,21 +84,10 @@ public static MetadataReference SystemThreadingTaskFacadeRef
             }
         }
 
-        protected bool PrintActualDiagnosticsOnFailure
-        {
-            set
-            {
-                _printActualDiagnosticsOnFailure = value;
-            }
-        }
+        protected bool PrintActualDiagnosticsOnFailure { private get; set; }
```

Comment:
By default, it will be `false`, when writing the test case, the user can set it to `true`. This way, the usage is simple.